worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/CreditStreamManager.java

```diff
@@ -55,21 +55,21 @@ public class CreditStreamManager {
   public CreditStreamManager(
       int minReadBuffers,
       int maxReadBuffers,
-      int threadsPerMountpoint,
+      int threadsPerMountPoint,
       int minBuffersToTriggerRead) {
     nextStreamId = new AtomicLong((long) new Random().nextInt(Integer.MAX_VALUE) * 1000);
     streams = JavaUtils.newConcurrentHashMap();
     activeMapPartitions = JavaUtils.newConcurrentHashMap();
     this.minReadBuffers = minReadBuffers;
     this.maxReadBuffers = maxReadBuffers;
-    threadsPerMountPoint = threadsPerMountpoint;
```

Comment:
so this is a bug?

Reply:
does it cause a real impact?

Reply:
We just use different variable names here to avoid this problem.

https://errorprone.info/bugpattern/InconsistentCapitalization

Reply:
Oh, I see